### PR TITLE
[15.0][MIG] commission, account_commission, sale_commission : Migration to 15.0

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
For reference, discussion session here [328](https://github.com/OCA/commission/issues/328)

### CODE REVIEW

**Comment by ValentinVinagre** [comment]
@pedrobaeza  is everything correct?

**Comment by pedrobaeza** [comment]
I will try to review ASAP. Why doing this in this fork?

**Comment by yostashiro** [comment]
Was intending to review this internally before making a PR in the OCA repo to reduce the noise (but just haven't got the time to do so).

@AungKoKoLin1997 Please go ahead create a PR in the OCA repo as talked this morning.

File: account_commission/wizards/wizard_settle.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        if self.settlement_type == "invoice":
+            return self.env["account.invoice.line.agent"].search(
+                [
+                    ("invoice_date", "<", date_to_agent),
+                    ("agent_id", "=", agent.id),
+                    ("settled", "=", False),
+                ],
+                order="invoice_date",
+            )
+        return super()._get_agent_lines
```
Comment (AungKoKoLin1997) [inline account_commission/wizards/wizard_settle.py:30]: @yostashiro The purpose of this code is we want to share this wizard for every source? When we choose settlement type to invoice, we will get commission data from invoice and will create settlement. If my thought is correct, can you please pay attention to the following conversation? Question(https://github.com/OCA/commission/issues/328#issuecomment-1253286983) Answer(https://github.com/OCA/commission/issues/328#issuecomment-1253288978)

**Comment by AungKoKoLin1997** [comment]
This PR is already merged in OCA repo and  I will close this one.

**Pull request closed without merging**